components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -151,6 +151,26 @@ spec:
 
           - name: TYK_DB_REDISADDRS
             value: {{ include "tyk-dashboard.redis_url" . | quote }}
+
+          - name: TYK_DB_REDISSSLINSECURESKIPVERIFY
+            value: "{{ .Values.global.redis.sslInsecureSkipVerify }}" 
+        
+          {{- if .Values.global.redis.useSSL }}
+          - name: TYK_DB_REDISSSLMAXVERSION
+            value: "{{ .Values.global.redis.sslMaxVersion }}"
+          - name: TYK_DB_REDISSSLMINVERSION
+            value: "{{ .Values.global.redis.sslMinVersion }}"  
+          {{- end}}  
+
+          {{- if .Values.global.redis.sslCAFile }}
```

Comment:
We should also check if cert/key file exists. Since sometimes user just provide cert and key without proving CA